test/commands/test_remote.rb

```diff
@@ -116,4 +116,42 @@ def test_remote_trim_with_tolerate
       assert_equal(1, remotes.all.count)
     end
   end
+
+  # @todo #329:30min Verify that the nodes that are being selected are
+  #  really the strongest ones. The strongest nodes are the ones with
+  #  the highest score.
+  def test_select_selects_the_strongest_nodes
+    skip
+  end
+
+  def test_select_respects_max_nodes_option
+    Dir.mktmpdir 'test' do |dir|
+      remotes = Zold::Remotes.new(File.join(dir, 'remotes.txt'))
+      zero = Zold::Score::ZERO
+      cmd = Zold::Remote.new(remotes: remotes, log: test_log)
+      (5000..5010).each do |port|
```

Comment:
@botanicus how do you feel about extracting this to a method?

Reply:
@ledestin I'm feeling like I seriously run out of time on this one. It's been going on for 9 days and I only get half an hour paid.